app/Exceptions/Handler.php

```diff
@@ -117,14 +117,11 @@ protected function unauthenticated($request, AuthenticationException $exception)
     public function sendEmail(Throwable $exception): void
     {
         try {
-            $content['message'] = $exception->getMessage();
-            $content['file'] = $exception->getFile();
-            $content['line'] = $exception->getLine();
-            $content['trace'] = $exception->getTrace();
-            $content['url'] = request()->url();
-            $content['body'] = request()->all();
-            $content['ip'] = request()->ip();
-            Mail::send(new ExceptionOccured($content));
+            $e = FlattenException::createFromThrowable($exception);
+            $handler = new HtmlErrorRenderer(true);
+            $css = $handler->getStylesheet();
+            $content = $handler->getBody($e);
+            Mail::send(new ExceptionOccured($content, $css));
```

Comment:
the exception error mail was not working so i did this.
But there is an issue with `jeremykenedy/laravel-email-database-log` package
`PDOException: SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'body'`